msal/src/Microsoft.Identity.Client/Features/DeviceCode/DeviceCodeResult.cs

```diff
@@ -1,29 +1,29 @@
-﻿//------------------------------------------------------------------------------
```

Comment:
I've no idea what happened to the heading comments here ?  I did not change anything there?